tflint crashed

This is what I've got

```
Panic: interface conversion: interface {} is int, not string
 -> 0: main.main.func1: /main.go(17)
 -> 1: runtime.call32: /asm_amd64.s(514)
 -> 2: runtime.gopanic: /panic.go(489)
 -> 3: runtime.panicdottype: /iface.go(172)
 -> 4: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalOutput).Eval: /eval.go(399)
 -> 5: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalVisitor).visit: /eval.go(207)
 -> 6: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalVisitor).(github.com/wata727/tflint/vendor/github.com/hashicorp/hil.visit)-fm: /eval.go(167)
 -> 7: github.com/wata727/tflint/vendor/github.com/hashicorp/hil/ast.(*Output).Accept: /output.go(23)
 -> 8: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalVisitor).Visit: /eval.go(167)
 -> 9: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.internalEval: /eval.go(145)
 -> 10: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.Eval: /eval.go(52)
 -> 11: github.com/wata727/tflint/evaluator.(*Evaluator).Eval: /evaluator.go(67)
 -> 12: github.com/wata727/tflint/evaluator.(*Evaluator).evalModuleAttr: /module.go(44)
 -> 13: github.com/wata727/tflint/evaluator.(*Evaluator).initModule: /module.go(24)
 -> 14: github.com/wata727/tflint/evaluator.NewEvaluator: /evaluator.go(35)
 -> 15: github.com/wata727/tflint/detector.NewDetector: /detector.go(88)
 -> 16: main.(*CLI).Run: /cli.go(116)
 -> 17: main.main: /main.go(30)
 -> 18: runtime.main: /proc.go(185)
 -> 19: runtime.goexit: /asm_amd64.s(2197)

TFLint crached... :(
Please attach an output log, describe the situation and version that occurred and post an issue to https://github.com/wata727/tflint/issues
```

version 0.4.0

not sure if I need to attach anything else to help debugging this

Thanks, @wata727.
Still doesn't work (version updated)

```
$ make lint
TF_VAR_datadog_api_key="qwe" tflint --debug
DEBUG:/loader.go:120 [INFO] Load environment...
DEBUG:/loader.go:128 [ERROR] open .terraform/environment: no such file or directory
DEBUG:/loader.go:131 [INFO] Load tfstate...
DEBUG:/loader.go:135 [ERROR] stat terraform.tfstate: no such file or directory
DEBUG:/loader.go:141 [INFO] Remote state detected
DEBUG:/loader.go:161 [INFO] Load tfvars...
DEBUG:/loader.go:164 [INFO] Load `terraform.tfvars`
DEBUG:/loader.go:166 [ERROR] stat terraform.tfvars: no such file or directory
DEBUG:/loader.go:191 [INFO] Load HCL file: `aws.tf`
DEBUG:/loader.go:53 [INFO] Load HCL file: `aws.tf`
DEBUG:/loader.go:191 [INFO] Load HCL file: `datadog.tf`
DEBUG:/loader.go:53 [INFO] Load HCL file: `datadog.tf`
DEBUG:/loader.go:191 [INFO] Load HCL file: `global.tf`
DEBUG:/loader.go:53 [INFO] Load HCL file: `global.tf`
DEBUG:/loader.go:191 [INFO] Load HCL file: `remote.tf`
DEBUG:/loader.go:53 [INFO] Load HCL file: `remote.tf`
DEBUG:/loader.go:191 [INFO] Load HCL file: `variables.tf`
DEBUG:/loader.go:53 [INFO] Load HCL file: `variables.tf`
Panic: interface conversion: interface {} is int, not string
 -> 0: main.main.func1: /main.go(17)
 -> 1: runtime.call32: /asm_amd64.s(514)
 -> 2: runtime.gopanic: /panic.go(489)
 -> 3: runtime.panicdottype: /iface.go(172)
 -> 4: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalOutput).Eval: /eval.go(449)
 -> 5: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalVisitor).visit: /eval.go(215)
 -> 6: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalVisitor).(github.com/wata727/tflint/vendor/github.com/hashicorp/hil.visit)-fm: /eval.go(175)
 -> 7: github.com/wata727/tflint/vendor/github.com/hashicorp/hil/ast.(*Output).Accept: /output.go(23)
 -> 8: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.(*evalVisitor).Visit: /eval.go(175)
 -> 9: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.internalEval: /eval.go(153)
 -> 10: github.com/wata727/tflint/vendor/github.com/hashicorp/hil.Eval: /eval.go(52)
 -> 11: github.com/wata727/tflint/evaluator.(*Evaluator).Eval: /evaluator.go(72)
 -> 12: github.com/wata727/tflint/evaluator.(*Evaluator).evalModuleAttr: /module.go(44)
 -> 13: github.com/wata727/tflint/evaluator.(*Evaluator).initModule: /module.go(24)
 -> 14: github.com/wata727/tflint/evaluator.NewEvaluator: /evaluator.go(39)
 -> 15: github.com/wata727/tflint/detector.NewDetector: /detector.go(89)
 -> 16: main.(*CLI).Run: /cli.go(116)
 -> 17: main.main: /main.go(30)
 -> 18: runtime.main: /proc.go(185)
 -> 19: runtime.goexit: /asm_amd64.s(2197)

TFLint crached... :(
Please attach an output log, describe the situation and version that occurred and post an issue to https://github.com/wata727/tflint/issues

make: *** [lint] Error 2
```

--- a/evaluator/evaluator_test.go
+++ b/evaluator/evaluator_test.go
@@ -74,6 +74,16 @@ variable "name" {
 			Result: "test",
 		},
 		{
+			Name: "completed integer variable",
+			Input: `
+variable "name" {
+    type = "string"
+    default = 1
+}`,
+			Src:    "${var.name}",
+			Result: "1",
+		},
+		{
 			Name: "completed list variable",
 			Input: `
 variable "name" {
@@ -104,6 +114,15 @@ variable "name" {
 }`,
 			Src:    "${var.name}",
 			Result: "test",
+		},
+		{
+			Name: "integer variable in missing type",
+			Input: `
+variable "name" {
+    default = 1
+}`,
+			Src:    "${var.name}",
+			Result: "1",
 		},
 		{
 			Name: "list variable in missing key",

--- a/evaluator/variable.go
+++ b/evaluator/variable.go
@@ -3,6 +3,8 @@ package evaluator
 import (
 	"reflect"
 
+	"fmt"
+
 	"github.com/hashicorp/hcl"
 	hclast "github.com/hashicorp/hcl/hcl/ast"
 	hilast "github.com/hashicorp/hil/ast"
@@ -99,7 +101,7 @@ func parseVariable(val interface{}, varType string) hilast.Variable {
 	case HCL_STRING_VARTYPE:
 		hilVar = hilast.Variable{
 			Type:  hilast.TypeString,
-			Value: val,
+			Value: fmt.Sprint(val),
 		}
 	case HCL_MAP_VARTYPE:
 		// When HCL map var convert(parse) to Go var,

--- a/evaluator/variable_test.go
+++ b/evaluator/variable_test.go
@@ -539,11 +539,11 @@ func TestParseVariable(t *testing.T) {
 						Value: map[string]hilast.Variable{
 							"test3": {
 								Type:  hilast.TypeString,
-								Value: 1,
+								Value: "1",
 							},
 							"test4": {
 								Type:  hilast.TypeString,
-								Value: 10,
+								Value: "10",
 							},
 						},
 					},

